src/components/PipelineRun/PipelineRunListView/__tests__/pipelinerun-actions.spec.tsx

```diff
@@ -1,3 +1,8 @@
+/* eslint-disable @typescript-eslint/require-await */
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+// These checks are disabled for this test.
```

Comment:
use // eslint-disable-next-line with a comment and specific rule instead of disabling rules in the file.

Reply:
individual comments have been added, please check again